app/scripts/controllers/landing.js

```diff
@@ -44,10 +45,26 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        $scope.filterUpdate();
       });
     }, 500, {leading: false});
 
-    $scope.dateFilter = function(event){
+    $scope.filterSearchResults = function(){
+      return _.filter($scope.searchResults, event => {
+        return dateFilter(event) && locationFilter(event);
+      });
+    };
+
+    $scope.filterUpdate = function(){
+      analytics.digitalData.searchTerm = $scope.searchVal;
+      analytics.digitalData.searchLocationFilter = $scope.eventFilters.locationName;
+      analytics.digitalData.searchDateFilter = $scope.eventFilters.date;
```

Comment:
@brenton05 I split these out into 2 variables. Let me know if you want me to name them something else or try to serialize them back into one variable (and what format you would want that in).